Generalize Attester Description (#142)
ietf-rats-wg/architecture: draft-ietf-rats-architecture.md

discussed could even be combined into even more complex implementations.

Claims are collected from Target Environments, as shown in {{twotypes-env}}.
That is, Attesting Environments collect the values and the information to be represented in Claims, by reading system registers and variables, calling into subsystems, taking measurements on code or memory and so on of the Target Environment.
Attesting Environments then format the claims appropriately, and typically
use key material and
cryptographic functions, such as signing or cipher algorithms, to
create Evidence.

> There is no limit to or requirement of places an Attesting Environment can exist, but they typically are in Trusted Execution Environments (TEE), embedded Secure Elements

Reviewer: ```suggestion
There is no limit to or requirement on the places an Attesting Environment can exist, but they typically are in Trusted Execution Environments (TEE), embedded Secure Elements
```

> There is no limit to or requirement of places an Attesting Environment can exists, but they typically are in Trusted Execution Environments (TEE), embedded Secure Elements

Reviewer 2: ```suggestion
There is no limit to or requirement of places an Attesting Environment can exist, but they typically are in Trusted Execution Environments (TEE), embedded Secure Elements
```

> There is no limit to or requirement on the places an Attesting Environment can exist, but they typically are in Trusted Execution Environments (TEE), embedded Secure Elements

Reviewer: ```suggestion
There is no limit to or requirement on the places that an Attesting Environment can exist, but they typically are in Trusted Execution Environments (TEE), embedded Secure Elements
```
